api/client/clientset/typed/compose/v1alpha3/compose_client.go

```diff
@@ -55,7 +55,7 @@ func setConfigDefaults(config *rest.Config) error {
 	gv := v1alpha3.SchemeGroupVersion
 	config.GroupVersion = &gv
 	config.APIPath = "/apis"
-	config.NegotiatedSerializer = serializer.DirectCodecFactory{CodecFactory: scheme.Codecs}
+	config.NegotiatedSerializer = serializer.WithoutConversionCodecFactory{CodecFactory: scheme.Codecs}
```

Comment:
Note to reviewer: This change was based on this: https://github.com/kubernetes/kubernetes/pull/76270